app/src/main/res/drawable/cat_animation.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<animation-list xmlns:android="http://schemas.android.com/apk/res/android"
+    android:oneshot="true">
```

Comment:
We need to play animation continuously not in one shot

Reply:
Thanks, already fixed